website/source/assets/stylesheets/_config.scss

```diff
@@ -0,0 +1,64 @@
+.config-reveal-label {
+  .config-reveal-trigger:not(:checked) + & {
+    color: #aaa;
+  }
+
+  .docs-info-icon {
+    &::before {
+      content: '\0024D8';
+      color: #AAA;
+      display: inline-block;
+      height: 1em;
+      margin-left: 0.25em;
+      width: 1em;
+    }
+
+    &:hover::before {
+      color: inherit;
+    }
+  }
+}
+
+.reveal-container,
+.config-reveal-container {
+  display: none;
+  margin-left: 20px;
+
+  &.active {
+    display: block;
+  }
+}
+
+.reveal-trigger {
+  align-items: center;
+  color: $sidebar-link-color-active;
+  cursor: pointer;
+  display: flex;
```

Comment:
Any IE boogeymen to look out for here? (I think mostly no since there's not `flex:` declarations).

Reply:
That's okay. If it can't flex then it will be full width like it is currently

Reply:
At least from the digital team perspective, we do not support browsers that don't support flexbox, so this is 👍 for me haha